buildSrc/src/main/groovy/io.micronaut.build.internal.grpc-base.gradle

```diff
@@ -0,0 +1,16 @@
+repositories {
+    mavenCentral()
+    maven {
+        url "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+        mavenContent {
+            snapshotsOnly()
+        }
+    }
```

Comment:
Why do we need the snapshot repo?

Reply:
we will need once we target 4.0.0-SNAPShOT